cli/commands/stack/command.go

```
@@ -8,14 +8,47 @@ import (
 
 const (
 	// CommandName stack command name.
-	CommandName = "stack"
-	generate    = "generate"
-	run         = "run"
+	CommandName          = "stack"
+	OutputFormatFlagName = "format"
+	OutputFormatEnvName  = "TERRAGRUNT_STACK_OUTPUT_FORMAT"
+	JsonFormatFlagName   = "json"
```

golangci-lint (1.62.2)

ST1003: const JsonFormatFlagName should be JSONFormatFlagName

(stylecheck)